Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -679,7 +679,9 @@ public struct ApolloCodegenConfiguration: Codable, Equatable {
   // MARK: - Properties
 
   /// Name used to scope the generated schema type files.
-  public let schemaName: String
+  @available(*, deprecated, renamed: "schemaNamespace")
+  public var schemaName: String { schemaNamespace }
```

Comment:
Can we move deprecated properties and initializers into an extension with `// MARK: - Deprecations` so they aren't in the way as much? It also give us an easy place to keep them so we can keep track of stuff to kill with the next major (breaking) version.